nservicebus/messaging/handling-responses-on-the-client-side.md

```diff
@@ -36,6 +36,29 @@ To access the response message through callbacks, the following code can be used
 
 DANGER: If the server process returns multiple responses, NServiceBus cannot know which response message will be the last. To prevent memory leaks, the callback is invoked only for the first response. Callbacks won't survive a process restart (common scenarios are a crash or an IIS recycle) as they are held in memory, so they are less suitable for server-side development where fault-tolerance is required. In those cases, [sagas are preferred](/nservicebus/sagas/).
 
+## NServiceBus 6: Handling responses in the context of the request
```

Comment:
i would split this into functionality headings.

eg   "where to get callback functionality: in v5 an lower it it build in, in v6 get the nuget"